requirements/dev.txt

```diff
@@ -15,7 +15,7 @@ diff-cover==1.0.2
 distlib==0.2.6            # via caniusepython3
 djangorestframework==3.6.3
 django-model-utils==3.0.0
-edx-i18n-tools==0.4.3
+#edx-i18n-tools==0.4.3
```

Comment:
We should just remove this one if we don't need i18n-tools in this repo. I don't see a reason we do need it.